install-dev.sh

```diff
@@ -0,0 +1,8 @@
+set -e
```

Comment:
Please add a valid file header and a shell shebang. `#!/bin/sh`

Reply:
Hi @feckert I am not sure if we need to add file headers in all files. License file is added in each OpenWISP module but I am not able to find a similar file header in other modules.